indigo/indigo/src/main/scala/indigo/shared/datatypes/RGBA.scala

```diff
@@ -82,7 +82,8 @@ object RGBA:
     }
 
   def fromHexString(hex: String): RGBA =
-    hex.trim match {
+    val hex = if hex.startsWith("#") then hex.substring(1).trim else hex.trim
```

Comment:
Shadowing produces a less readable code, imo. You could also use `tail` here instead of `substring(1)`, which could fail when the string is empty (not the case because it's nested within that `if`)

Reply:
There is also an established pattern here for this. Now I look at it I am wondering why on earth I didn't just use a regex, but nonetheless it would be good if all cases were extracted in the same manner.